corehq/apps/prototype/static/prototype/js/example/knockout_pagination_webpack.main.js

```diff
@@ -0,0 +1,70 @@
+// base level imports need to figure out how to always include these
+
+//need to figure out gettext mapping
+import 'hqwebapp/js/bootstrap5/common';
```

Comment:
Can we import any HQ module here? I'm asking because `common` and `base_main` aren't typical modules (at least not in a local environment). For an example, can you import initial_page_data? Matt had to do some kind of non-ideal workaround for this.

Reply:
...nevermind, you import initial_page_data below, grand

Reply:
although I am curious why that didn't work for Matt - @mjriley what's the difference here?

Reply:
I might have just assumed it wouldn't work and didn't try? I like Biyeun's approach here to focus more on the webpack side, but I was kind of limiting my investigation into Webpack to focus on just the React stuff

Reply:
i made a lot of changes to the config file that opened it up for referencing hq paths the way we are used to with requirejs :)